pycmds/hardware/filters.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, *args, **kwargs):
         self.motor = yaqc.Client(self.yaqd_port, host=self.yaqd_host)
         self.motor_units = self.motor.get_units()
         if self.motor_units == "deg":
-            self.motor_units = "deg_filter_wheel"
+            self.motor_units = "deg"
```

Comment:
This if can go away,  was only there for delays because `mm` is different in wt.units from `mm_delay` (factor of 2, because delay is assumed retroreflective, which doubles the distance for the light)

As it stands, this is a no-op